src/Ledger/Deleg.lagda

```diff
@@ -114,9 +120,10 @@ module _ (open PParams) where
 data _⊢_⇀⦇_,DELEG⦈_ : DelegEnv → DState → DCert → DState → Set where
   DELEG-delegate :
     d ≡ requiredDeposit pp mv ⊔ requiredDeposit pp mc
+    → mc ∈ mapˢ just (dom pools)
     ────────────────────────────────
-    pp ⊢  ⟦ vDelegs , sDelegs , rwds ⟧ᵈ ⇀⦇ delegate c mv mc d ,DELEG⦈
-          ⟦ insertIfJust c mv vDelegs , insertIfJust c mc sDelegs , rwds ⟧ᵈ
+    ⟦ pp , pools ⟧ᵈᵉ ⊢  ⟦ vDelegs , sDelegs , rwds ⟧ᵈ ⇀⦇ delegate c mv mc d ,DELEG⦈
+                       ⟦ insertIfJust c mv vDelegs , insertIfJust c mc sDelegs , rwds ⟧ᵈ
```

Comment:
```suggestion
                        ⟦ insertIfJust c mv vDelegs , insertIfJust c mc sDelegs , rwds ⟧ᵈ
```

(alignment)

Reply:
Thanks for catching these, my font isn't perfectly monospace so I sometimes get it off by one. Probably should fix that some time...